### PR TITLE
Add 'New update available' dialog to the electron version

### PR DESCRIPTION
**Pull request by ert78gb**
Suggest to accept this PR after #298

### CODE REVIEW

File: electron/src/store/actions/app-update.action.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const PREFIX = '[app-update] ';
+
+// tslint:disable-next-line:variable-name
+export const ActionTypes = {
+    UPDATE_AVAILABLE: type(PREFIX + 'update available'),
+    UPDATE_APP: type(PREFIX + 'update app'),
+    DO_NOT_UPDATE_APP: type(PREFIX + 'do not update app'),
+    UPDATE_DOWNLOADED: type(PREFIX + 'update downloaded'),
+    UPDATEING: type(PREFIX + 'updating')
```
Comment (attilacsanyi) [inline electron/src/store/actions/app-update.action.ts]: typo ;-) `UPDATING`

Comment (ert78gb) [inline electron/src/store/actions/app-update.action.ts]: thx, fixed

**Comment by ert78gb** [comment]
Pls wait for the review. I am working on the update settings form I will it add during the day

**Comment by ert78gb** [comment]
https://github.com/electron-userland/electron-builder/issues/1709 block me

**Comment by ert78gb** [comment]
The guys solved the problem with electron builder so you could accept this PR.

The usb dependency library will be install as node module. Does need to copy it directly into the vendor folder? I think no, but I don't like to delete until you will not approve.

**Pull request merged**